### PR TITLE
Action toolbar approve

### PR DESCRIPTION
**Pull request by berndvdveen**
- Event message styling
- Correctly bind approve function to action toolbar output
- Add start delivery action
- Update ng Jhipster dependency
- Return extended DTO for Send for Review action

### CODE REVIEW

File: podium-gateway/src/main/webapp/app/shared/event/podium-event-message.component.ts [hunk 1]
```diff
@@ -35,17 +35,18 @@ export class PodiumEventMessageComponent implements OnInit {
...
 
     ngOnInit() {
         this.findLastHistoricMessageEventForCurrentStatus();
     }
 
     findLastHistoricMessageEventForCurrentStatus() {
-        let events: PodiumEvent[] = this.request.historicEvents;
-        let lastHistoricEvent = this.request.historicEvents[events.length - 1];
-        this.lastEvent = lastHistoricEvent;
+        let messageEvents = this.request.historicEvents.filter((event) => {
```
Comment (rnugraha) [inline podium-gateway/src/main/webapp/app/shared/event/podium-event-message.component.ts]: Can we have unit testing for this

File: podium-gateway/src/main/webapp/app/shared/request/action-bars/request-action-toolbar/request-action-toolbar.component.html [hunk 2]
```diff
@@ -138,6 +138,28 @@
...
                 </div>
             </div>
+
+            <!-- Approved -->
+            <div class="action-container" *ngIf="isStatus(requestStatus.Review)">
+
+                <!-- Researcher actions -->
+                <div *ngIf="isRequestingResearcher()">
+                    <div class="alert alert-info" jhiTranslate="request.toolbar.approved">
+                        Your request has been approved and is pending start of delivery by the organisation coordinator.
```
Comment (rnugraha) [inline podium-gateway/src/main/webapp/app/shared/request/action-bars/request-action-toolbar/request-action-toolbar.component.html]: Pls have max 120 chars

File: podium-gateway/src/main/webapp/app/request/main-detail/progress-bar/request-progress-bar.component.ts [hunk 3]
```diff
@@ -78,14 +79,14 @@ export class RequestProgressBarComponent {
...
+     * Check whether the request is terminated
      * All closed states return an order of -1
      *
      * @param request the current request
      * @returns {boolean} true if the request status is closed
      */
-    isClosed(request: RequestBase): boolean {
-        return this.getRequestStatusOrder(request) === -1;
+    isTerminatedReview(request: RequestBase): boolean {
+        return this.getRequestStatusOrder(request) === -1 && request.requestReview.decision === RequestReviewDecision.Rejected;
```
Comment (rnugraha) [inline podium-gateway/src/main/webapp/app/request/main-detail/progress-bar/request-progress-bar.component.ts]: pls format to max 120 chars and I would expect this function unit tested

File: podium-gateway/src/main/webapp/app/shared/request/request-review-panel/request-review-panel.component.html [hunk 4]
```diff
@@ -1,32 +1,34 @@
...
+            <td>
+                <small>{{feedback.summary}}</small>
+            </td>
+        </tr>
+        </tbody>
+        <tbody *ngIf="!lastReviewFeedback">
+        <tr>
+            <td colspan="4" class="no-results">
+                <div class="alert alert-info">
+                    No review found
```
Comment (rnugraha) [inline podium-gateway/src/main/webapp/app/shared/request/request-review-panel/request-review-panel.component.html]: translation needed?

Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/shared/request/request-review-panel/request-review-panel.component.html]: I dont know ;) its your code. Ill be sure to add it

**Pull request merged**